modules/iqtree/iqtree.nf

```diff
@@ -1,5 +1,5 @@
 process IQTREE {
-    label 'cpu_8_memory_16'
+    label 'cpu_8_memory_6'
```

Comment:
⚠️  This will cause `iqtree` to fail with larger datasets, across different platforms.